ext/dom/tests/bug47847.phpt

```diff
@@ -1,27 +0,0 @@
---TEST--
-Bug #47847 (importNode loses the namespace of an XML element)
---EXTENSIONS--
-dom
---FILE--
```

Comment:
Possibly keep the tests but just mark them as XFAIL with a link to this PR to explain rationale for the revert?